Inline documentation for iteritems, dropna, drop, get

shril opened the pull request:
Inline documentation for drop function in frame.py file.

shril commented:
Hi @rxin , have a look at this.
I am working on the inline docs of other functions in frame.py file.

Happy to address any issues regarding my changes.

rxin commented on databricks/koalas/frame.py:
```diff
@@ -881,8 +881,47 @@ def unique(self):
         return DataFrame(spark.DataFrame(sdf._jdf.distinct(), sdf.sql_ctx), self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def drop(self, labels, axis=0, errors='raise'):
```
cc @thunterdb why do we have "errors" parameter that seem unused?

HyukjinKwon replied:
Seems like just wanted to match the signature (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html#pandas.DataFrame.drop) Let's remove.

shril replied:
Removing it.

rxin commented on databricks/koalas/frame.py:
```diff
@@ -881,8 +881,47 @@ def unique(self):
+        ----------
+        labels : single label or list-like
+            Column labels to drop.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```
say something about only axis = 1 is supported at the moment.

shril replied:
Added the information.

HyukjinKwon replied:
I think we should fix this doc.

rxin commented:
This looks great. For the next pull request, feel free to do more than one function.

shril commented:
@rxin I'll push the documentation of other functions as well in this PR itself.
I little caught up.

shril commented:
Hi @rxin , I have added documentation for several functions in this PR.

I was keeping a track of documentation of functions left in frame.py. Issue #152 
Unchecked ones are left.

- [x] to_html
- [x] isnull
- [x] notnull
- [x] toPandas
- [ ] assign
- [x] dropna
- [x] drop
- [x] get
- [ ] pipe
- [x] iteritems

HyukjinKwon commented on databricks/koalas/frame.py:
```diff
@@ -163,8 +163,44 @@ def corr(self, method='pearson'):
+        Iterates over the DataFrame columns, returning a tuple with
+        the column name and the content as a Series.
+
+        Yields
```
Let's make this API yield or fix the doc.

shril replied:
I am fixing the doc for now.

HyukjinKwon commented on databricks/koalas/frame.py:
```diff
@@ -691,8 +727,88 @@ def loc(self):
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```
I think we don;t support `columns` yet. Let's take out.

shril replied:
Doing it.

shril commented:
Hi @HyukjinKwon,

- Changed the docs for `dropna`, which only supports axis=0 at the moment.
- Changed the docs for `drop`, which only supports axis=1 at the moment.
- Changed from yields to returns in the documentation for `iteritems`.
- Removed error parameter from the `drop` function.

shril commented:
Hi @HyukjinKwon, I think my PR contains a large number of commits.
Should I fork another branch from master and transfer the changes to a single commit?

rxin commented:
It's ok when we merge into master we squash all commits into a single one.

garawalid commented on databricks/koalas/frame.py:
```diff
@@ -691,8 +727,86 @@ def loc(self):
+        --------
+        DataFrame.drop : Drop specified labels from columns.
+        DataFrame.isnull: Indicate missing values.
+        DataFrame.notnull : Indicate existing (non-missing) values.
```
We don't have `notnull()`.
You can replace it by `notna()`.

rxin replied:
We do have notnull, don't we? https://github.com/databricks/koalas/blob/master/databricks/koalas/frame.py#L535

shril replied:
Line 571 is `notnull()` in frame.py.
Please correct me if I am making any mistake here.

garawalid replied:
I saw the documentation. You re right

rxin commented:
Thanks. I've merged this in master.

HyukjinKwon commented:
Nice doc change. Thanks, @shril
